docs/reference/multisite.rst

```diff
@@ -100,38 +80,16 @@ The last action is to configure the ``sonata_page`` section as:
 Host and Path By Locale Strategy
 --------------------------------
 
-This strategy requires a dedicated ``RequestFactory`` object. So you need to alter the
-front controller to use the one provided by the ``PageBundle``.
-
-To do so, open ``index.php`` file and change the following parts::
-
-    // public/index.php
-
-    use App\Kernel;
-    use Symfony\Component\ErrorHandler\Debug;
-    use Sonata\PageBundle\Request\RequestFactory; # before: use Symfony\Component\HttpFoundation\Request;
-
-    require dirname(__DIR__).'/config/bootstrap.php';
-
-    if ($_SERVER['APP_DEBUG']) {
-        umask(0000);
-
-        Debug::enable();
-    }
+This strategy requires a dedicated runtime. So you need to modify your
+composer.json and declare ``SonataPageRuntime`` runtime.
```

Comment:
on the line 78

```yaml
 sonata_page:
        multisite: host_with_path
```

please do not remove this form the doc, it is really important to make this work.
It was the issue that I have, if I just use runtime it won't work. there are part of the code doing some logical with this parameter.

I guess it can be avoided in the future, but for  the current solution this config is needed ;)

Reply:
i never intended to do this as we need this to setup all services.